db/migrate/20230713045918_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.string :Bio
+      t.integer :PostsCounter
```

Comment:
- [ ] Kindly follow Ruby convention for data attributes, `Name` should be `name`, `Photo` should be `photo`, `Bio` should be `bio`, `PostsCounter` should be `posts_counter`

Reply:
Attributes updated